poc_config/openmrs/i18n/common/locale_en.json

```diff
@@ -110,5 +111,11 @@
     "CLINIC_PATIENT_WEIGHT": "Weight (kg)",
     "CLINIC_PATIENT_HEIGHT": "Height (cm)",
     "CLINIC_PATIENT_BMI": "BMI",
-    "LOGIN_LABEL_LOGGEDIN_KEY": "Logged in as"
+    "LOGIN_LABEL_LOGGEDIN_KEY": "Logged in as",
+    "THERAPEUTIC_REGIME_ADULTS" : "The sugested line is TDF/3TC/EFV",
```

Comment:
Isn't this supposed to be in  poc_config/openmrs/i18n/**clinic**/locale_en.json?